README.md

```diff
@@ -0,0 +1,20 @@
+# Support teacher training publishers
+
+This app is used by the user support team to help publishers with the Pubish teacher training courses service.
+
+## Setting up the app in development
+
+1. Create a Postgresql user:
```

Comment:
I really don’t like the idea of of this in development. Can we just go with a vanilla setup and only set username in production?

Reply:
Yeah no worries – I've updated the PR with this.